modules/material/locales/en/LC_MESSAGES/material.po

```diff
@@ -137,6 +137,9 @@ msgstr "USB key icon"
 msgid "{mfa:webauthn_instructions}"
 msgstr "You may now insert your security key and press its button."
```

Comment:
I think this text may be unnecessary at best, and incorrect at worst. There are other types of webauthn keys besides a "security key" that should be inserted and touched. The browser does a better job at guiding the user. @forevermatt what do you think?

Reply:
We certainly need to update the wording, but I don't know what to change it to, nor whether now is the time to do that. Feel free to use your judgment on this one, based on how it looks in an actual browser.

Reply:
Since there is a delay as the browser prompts for the key it might be nice to have something. Maybe "You may now use your passkey"